cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -11,23 +11,27 @@
 
    See [GHE]/flapjack/Modules-V1/wiki/50-50-Image-&-Text
 
-   settings:              An object with the following options for settings.
-   settings.title:        A string containing title of module
-   settings.image_path:   A string containing the url for the image
-   settings.image_alt:    A string containing alt text for screenreaders
+   settings:               An object with the following options for settings.
+   settings.title:         A string containing title of module
+   settings.image_path:    A string containing the url for the image
+   settings.image_alt:     A string containing alt text for screenreaders
    settings.is_widescreen: Boolean if the image should be widescreen ratio.
-                          Will be 4:3 ratio by default.
-   settings.description:  A string for the description under the image
-   settings.is_button:    Boolean if the link should be a button. If false/blank,
-                          the button will remain a plain link.
-   settings.link_url:     A string for the URL of the link/button
-   settings.link_text:    A string for the text of the link/button call to action
+                           Will be 4:3 ratio by default.
+   settings.description:   A string for the description under the image
+   settings.is_button:     Boolean if the link should be a button. If false/blank,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/f3595a300baf0a5429bfe71135bc596f0c866594